README.md

```diff
@@ -24,6 +24,23 @@ yarn kafka:start
 yarn start
 ```
 
+## Project Structure
+
+To understand the overall architecture of the project, please review our [architecture documentation](docs/ARCHITECTURE.md).
+
+The root structure is as follows:
+
+```
+- docs     // Project documentation
+- services // Convenience containers for various external services (e.g. Kafka)
+- src      // The primary code location itself
+| - components // Architectural components
+| - constants  // Constants used throughout the project
+| - lib        // Supporting code (e.g. utilities, DB singletons, etc)
+| - scripts    // Supporting scripts
+| - tests      // System-wide / integration tests
+```
```

Comment:
Are we going to have a `/src/models`, or were you thinking that would be in `/src/lib/models`? We don't need to mock it out in advance, just curious.

Reply:
Open to talking through it when the time comes, but I think it should be `/src/models`